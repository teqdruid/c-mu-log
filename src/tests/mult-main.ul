//main(a,b);
//@main("asdf");
main(dfsdf)
{
wall(4,5);
}



 
