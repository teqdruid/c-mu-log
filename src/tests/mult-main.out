
 ^^^  Solution  ^^^ 


 ^^^  Solution  ^^^ 

No more solutions
